doc/src/contributing/new-contributors-guide/dev-setup.md

```diff
@@ -175,7 +175,7 @@ You may want to take advantage of using virtual environments to isolate your dev
 If you use `conda`, you can use it to create a virtual environment:
 
 ```bash
-$ conda create -n sympy-dev python=3 mpmath flake8
+$ conda create -n sympy-dev python=3 mpmath flake8 pytest -c conda-forge hypothesis
```

Comment:
Is conda-forge required to install these? If so, the -c conda-forge should go at the beginning of the command. Otherwise, I would just omit it.

Reply:
We should probably also add `pytest hypothesis` to this command as well https://github.com/sympy/sympy/pull/25468/files#diff-1d362eb2450c8a795bf7dbcb3b67dd27057713c0dc9347a85aeedfba69b7cfe0R187

Reply:
It seems that people have trouble with `conda install hypothesis` and per the documentation https://anaconda.org/conda-forge/hypothesis , it seems like the proper way.

Reply:
@asmeurer 
1) if its installing via  `pip install -e .`, wouldn't it be better to just add pytest and hypothesis in `sympy/setup.py` and not change that part of the documentation. 
2) so `conda create -n sympy-dev -c conda-forge hypothesis python=3 mpmath flake8 pytest`?

Reply:
> if its installing via pip install -e ., wouldn't it be better to just add pytest and hypothesis in sympy/setup.py and not change that part of the documentation.

Yes. I think there is some way to specify development dependencies, so that something like `pip install sympy[dev]` (or `pip install .[dev]` (??)) installs them.

Reply:
I guess it's probably true that the hypothesis people would want people to use conda-forge, since the defaults channel likely doesn't update often enough given their release cadence. Although maybe @honno can confirm this.

Reply:
> I guess it's probably true that the hypothesis people would want people to use conda-forge, since the defaults channel likely doesn't update often enough given their release cadence.

Yeah pretty sure that was the thinking.